src/runtime/tests/arc-test.ts

```diff
@@ -808,10 +803,7 @@ describe('Arc', () => {
     //assert.strictEqual(slotComposer.slotsCreated, 1);
   });
 
-  it('serialization roundtrip preserves data for volatile stores', async function() {
-    if (Flags.useNewStorageStack) {
-      this.skip();
-    }
+  it.skip('serialization roundtrip preserves data for volatile stores', async () => {
```

Comment:
could you add a comment on why this is skipped?

Reply:
I don't actually know, sorry. It was skipped before so I kept it skipped in this patch.